src/cpp/session/modules/SessionFind.cpp

```diff
@@ -1095,6 +1127,30 @@ struct ReplaceOptions
    const std::string replacePattern;
 };
 
+void addDirectoriesToCommand(
+   bool packageFlag, const FilePath& directoryPath, shell_utils::ShellCommand* pCmd)
+{
+   // TODO: review if switching to EscapeFilesOnly is necessay
```

Comment:
We don't typically commit TODOs since we have no mechanism for ensuring that they get resolved. If the PR contains unfinished work items, open an issue to track them instead (and optionally refer to it in a comment).

Reply:
I suggested to add a TODO or a comment here because it works as-is and whatever the outcome of the review, it should continue to work. The only benefit of the change is removing a single line of code, which didn't seem significant enough to warrant a git issue. I just though it would be good to have a note in the code that if someone happens to be looking here and they happen to know the answer, they can make this change or update the comment.

Reply:
Makes sense but I think it's unlikely in practice. We usually use TODO as a way to help flag things that need to be resolved before commit, so having TODO in committed code adds noise. Let's leave the comment and rephrase it without TODO.

Reply:
Ok, I updated the comment